tests/test_controlbyweb_web_relay.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+from its_preselector.controlbyweb_web_relay import ControlByWebWebRelay
+from requests import codes
+from requests import Response
+import xml.etree.ElementTree as ET
```

Comment:
If you decide to make the switch to `defusedxml`, this is another place to swap out `xml.etree.ElementTree`

Reply:
done.